src/main/java/uk/gov/pay/adminusers/model/Service.java

```diff
@@ -1,27 +1,58 @@
 package uk.gov.pay.adminusers.model;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static uk.gov.pay.adminusers.app.util.RandomIdGenerator.newId;
+import static uk.gov.pay.adminusers.app.util.RandomIdGenerator.randomInt;
 
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class Service {
 
     public static String DEFAULT_NAME_VALUE = "System Generated";
 
-    private Integer id;
+    public static final String FIELD_SERVICE_NAME = "name";
 
+    private Integer id;
+    private String externalId;
     private String name = DEFAULT_NAME_VALUE;
+    private List<Link> links = new ArrayList<>();
+
+    public static Service from() {
+        return from(DEFAULT_NAME_VALUE);
+    }
+
+    public static Service from(String name) {
+        return from(randomInt(), newId(), name);
```

Comment:
@DanailMinchev it doesn't have to be as all it requires is a unique string.
However, changing all usages of `newId()` to `randomUuid()` for consistency purposes. Except for OTP_KEY.